.github/workflows/release.yaml

```diff
@@ -98,7 +98,9 @@ jobs:
           for file in *; do
             if [ -f "$file" ]; then
               new_name=$(echo "$file" | tr ' ' '-')
-              mv "$file" "$new_name"
+              if [ "$new_name" != "$file" ]; then # Skip files with no spaces
```

Comment:
Same result can be achieved with `-n` flag.
Docs: 
-n, --no-clobber
do not overwrite an existing file